PWGHF/D2H/Tasks/taskFlowCharmHadrons.cxx

```diff
@@ -19,19 +19,23 @@
 #include "Framework/HistogramRegistry.h"
 #include "Framework/runDataProcessing.h"
 
+#include "CCDB/BasicCCDBManager.h"
+
```

Comment:
I cannot make the suggestion here, but the line `#include "CCDB/BasicCCDBManager.h"` should go before the line `#include "Framework/AnalysisTask.h"`.